test/files/als006/README.md

```diff
@@ -0,0 +1,6 @@
+| File name | Expected result | Error | Description |
+|-------|---------|--------|-------|
+| pass-als006-sectioned-solid-horizontal.ifc| success |    | |
+| fail-als006-sectioned-solid-horizontal-wrong-identifier.ifc| fail | On instance #200001=IfcShapeRepresenta...,(#54)) the following invalid value for RepresentationIdentifier has been found: Footprint| The value of attribute Representation must be Axis, but is Footprint instead  |
+| fail-als006-sectioned-solid-horizontal-wrong-item.ifc| fail |The instance #20001=IfcShapeRepresentation(#1001, 'Axis', 'Curve2D',(#79)) expected type 'IfcCompositeCurve or IfcIndexedPolycurve or IfcPolyline' for the attribute Items, but found #79=IfcGradientCurve((#8..#54,$) | On entity #20001=IfcShapeRepresentation(#1001, 'Axis', 'Curve2D',(#79)). The type of attribute Items must be IfcCompositeCurve or IfcIndexedPolycurve or IfcPolyline, but found IfcGradientCurve instead |
+| fail-als006-sectioned-solid-horizontal-wrong-type.ifc| fail | On instance #20001=IfcShapeRepresenta...,(#54)) the following invalid value for RepresentationType has been found: Curve3D |The value of attribute RepresentationType must be Curve2D, but found Curve3D instead |
```

Comment:
This table seems to have more information than the one for ALS005 (#76). Can this be unified?

Meta: are there any guidelines, how these tables should look like?

Reply:
The guidelines are in : 
https://github.com/buildingSMART/ifc-gherkin-rules/blob/main/features/README.md

A template of how these tables should look like specifically in https://github.com/buildingSMART/ifc-gherkin-rules/blob/main/features/README.md#table-template-for-unit-test-files

The error column should display the error message that is generated by the validation service. In 'Description' there's the option to add something specific to that file if necessary. But ideally, the error message that is generated automatically should be descriptive enough that this extra information is not necessary. 

The table in als005 is generated programmatically with the script in ; https://github.com/buildingSMART/ifc-gherkin-rules/pull/83